Samples/london-no2.py

```diff
@@ -12,5 +16,5 @@
 
 dataset = Dataset(subjects=[subjects], datasources=[datasources], fields=[lvf])
 recipe = Recipe(dataset=dataset)
-recipe.build_recipe(output_location='Desktop/london-no2.json')
-recipe.run_recipe(tombolo_path='Desktop/UptodateProject/TomboloDigitalConnector', output_path='Desktop/london-no2.geojson')
+recipe.build_recipe(output_location=recipe_output_location)
+recipe.run_recipe(tombolo_path=tombolo_path, output_path=model_output)
```

Comment:
On Jupyter notebook i run the below and it hangs with no feedback on the console

%run "C:\Users\Jupiter\Desktop\digital-connector-python\recipe.py"

subject_geometry = Subject(subject_type_label='localAuthority',provider_label='uk.gov.ons',
                  match_rule=Match_Rule(attribute_to_match_on="name", pattern="London%"))

localAuthority = Datasource(importer_class='uk.org.tombolo.importer.ons.OaImporter',
                            datasource_id='localAuthority')

dataset = Dataset(subjects=[subject_geometry], fields=[],
                  datasources=[localAuthority])

recipe = Recipe(dataset)
recipe.build_recipe(output_location="test.json",
                   console_print=False)

dc_dir = 'C:\\Users\\Jupiter\Desktop\\TomboloDigitalConnector'
recipe.run_recipe(tombolo_path=dc_dir,
                  output_path = 'C:\\Users\\Jupiter\Desktop\\local_authorities.json')